integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -1167,63 +900,64 @@ private void createNewCluster() {
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Waiting for managed server pod {0} to be ready in namespace {1}",
-          managedServerPodNamePrefix + i, domainNamespace1);
-      checkPodReady(managedServerPodNamePrefix + i, domainUid, domainNamespace1);
+          managedServerPodNamePrefix + i, domainNamespace3);
+      checkPodReady(managedServerPodNamePrefix + i, domainUid, domainNamespace3);
     }
   }
 
-  private void scaleDomainAndVerifyCompletedEvent(int replicaCount, ScaleAction testType, boolean verify) {
+  private void scaleDomainAndVerifyCompletedEvent(int replicaCount, ScaleAction testType,
+                                                  boolean verify, String namespace) {
     OffsetDateTime timestamp = now();
     logger.info("Updating domain resource to set the replicas for cluster " + cluster1Name + " to " + replicaCount);
-    int countBefore = getDomainEventCount(domainNamespace1, domainUid, DOMAIN_COMPLETED, "Normal");
+    int countBefore = getDomainEventCount(namespace, domainUid, DOMAIN_COMPLETED, "Normal");
     V1Patch patch = new V1Patch("["
         + "{\"op\": \"replace\", \"path\": \"/spec/clusters/0/replicas\", \"value\": " + replicaCount + "}" + "]");
-    assertTrue(patchDomainCustomResource(domainUid, domainNamespace1, patch, V1Patch.PATCH_FORMAT_JSON_PATCH),
+    assertTrue(patchDomainCustomResource(domainUid, namespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH),
         "Failed to patch domain");
     int serverNumber = replicaCount + 1;
 
     switch (testType) {
       case scaleUp:
-        checkPodReady(managedServerPodNamePrefix + replicaCount, domainUid, domainNamespace1);
+        checkPodReady(managedServerPodNamePrefix + replicaCount, domainUid, namespace);
         break;
       case scaleDown:
-        checkPodDeleted(managedServerPodNamePrefix + serverNumber, domainUid, domainNamespace1);
+        checkPodDeleted(managedServerPodNamePrefix + serverNumber, domainUid, namespace);
         break;
       case reset:
-        checkPodReady(managedServerPodNamePrefix + replicaCount, domainUid, domainNamespace1);
-        checkPodDeleted(managedServerPodNamePrefix + serverNumber, domainUid, domainNamespace1);
+        checkPodReady(managedServerPodNamePrefix + replicaCount, domainUid, namespace);
+        checkPodDeleted(managedServerPodNamePrefix + serverNumber, domainUid, namespace);
         break;
       default:
     }
 
     if (verify) {
       logger.info("Verify the DomainCompleted event is generated after " + testType);
       checkEventWithCount(
-          opNamespace, domainNamespace1, domainUid, DOMAIN_COMPLETED, "Normal", timestamp, countBefore);
+          opNamespace, namespace, domainUid, DOMAIN_COMPLETED, "Normal", timestamp, countBefore);
     }
   }
 
-  private void scaleDomain(int replicaCount) {
-    scaleDomainAndVerifyCompletedEvent(replicaCount, ScaleAction.reset, false);
+  private void scaleDomain(int replicaCount, String namespace) {
```

Comment:
this method seems to be never used in this class

Reply:
removed